clang/test/CodeGenObjC/direct-method-nil-check-thunk-vararg.m

```diff
@@ -0,0 +1,62 @@
+// RUN: %clang -fobjc-emit-nil-check-thunk      \
+// RUN:   -target arm64-apple-darwin -fobjc-arc \
+// RUN:   -O0 -S -emit-llvm %s -o - | FileCheck %s
+
+
+#include <stdarg.h>
+
+int vprintf(const char * restrict format, va_list ap);
+#define NULL ((void *)0)
```

Comment:
According to https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/vprintf-vprintf-l-vwprintf-vwprintf-l?view=msvc-170 you need to import `stdio.h`, `stdarg.h` and `varargs.h` for UNIX V compatibility (if you want to try… I am not sure `va_list` is going to be visible without `varargs.h`, but the transitive headers are always a problem).

Reply:
> you need to import `stdio.h`

Windows was not happy about `#include <stdio.h>` saying it couldn't find it. That's why I only included necessary declaration.